src/components/main.js

```diff
@@ -267,8 +269,10 @@ class Main extends React.Component {
     this.state.session.setPage(page);
   }
 
-  handleMenuChange({key}) {
-    this.setState({ menuItem: key })
+  handleMenuChange ({ key }) {
+    const stateUpdate = { menuItem: key }
+    if (this.isMobile()) stateUpdate.collapsed = true
```

Comment:
What is the purpose of this line? You've already set `collasped: true` on L74

Reply:
When the user taps on a menu item, this line collapses the menu for them. 

L74 collapses the menu for mobile uses when the Main component is mounted, so because it doesn't unmount when the content changes, L74 doesn't fire again.